lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1799,8 +1799,11 @@ getNetworkParameters
     -> Handler ApiNetworkParameters
 getNetworkParameters (_block0, np, _st) nl = do
```

Comment:
Just a thought: Maybe we should call this `genesisNp` or something.

Reply:
Yes, good thought.